phpcs.xml.dist

```diff
@@ -20,10 +20,10 @@
 	<file>.</file>
 
 	<!-- Check for PHP cross-version compatibility. -->
-	<config name="testVersion" value="7.4-"/>
+	<config name="testVersion" value="8.2-"/>
 	<rule ref="PHPCompatibilityWP"/>
 
-	<config name="minimum_supported_wp_version" value="6.0"/>
+	<config name="minimum_supported_wp_version" value="8.0"/>
```

Comment:
There is no WP 8.0 (yet) 😄 

The latest 3 major versions should be checked against, so at the time of writing that would be 6.3.

Reply:
I confused this with PHP version :D